onnxruntime/core/optimizer/bias_dropout_fusion.cc

```diff
@@ -74,6 +74,26 @@ void FuseResidualAddIfAny(Graph& graph, const Node& dropout_node,
   }
 }
 
+static bool IsSameShape(const TensorShapeProto& shape1, const TensorShapeProto& shape2) {
```

Comment:
is "match &= ONNX_NAMESPACE::operator==(input1_shape->dim(i), input2_shape->dim(i));" doing the similar thing?